src/libs/ReportUtils.ts

```diff
@@ -1439,9 +1439,10 @@ function canDeleteReportAction(reportAction: OnyxEntry<ReportAction>, reportID:
             return false;
         }
 
+        const linkedReport = isChatThread(report) ? getReport(report?.parentReportID) : report;
```

Comment:
```suggestion
        const linkedReport = isThreadFirstChat(reportAction, reportID) ? getReport(report?.parentReportID) : report;
```
Since this is about deletion of the reportAction I think checking if the reportAction is a parent message is better than checking if the report is a thread. WDYT?

Reply:
makes sense